src/widget/wtracktableview.cpp

```diff
@@ -837,6 +845,9 @@ void WTrackTableView::dropEvent(QDropEvent * event) {
 }
 
 QModelIndexList WTrackTableView::getSelectedRows() const {
+    if (getTrackModel() == nullptr) {
+        return {};
+    }
```

Comment:
same here

Reply:
I installed this as some sort of pre-DEBUG_ASSERT exit in case someone uses getSelectedRows() without verifying there's a model. (see my explanation above)

Reply:
yeah keep this one then. Though I'm undecided on whether it should be a debug assert as well...